### DISCUSSION TITLE
Having systemwide performance/stability issues when streamdeck-linux-gui is running

### DISCUSSION THREAD

**Discussion by snek-the-great** [streamdeck-linux-gui/streamdeck-linux-gui#222] [Q&A]
Hi,
I'm writing in to ask if there's any way to improve system performance/stability when the app is running. I've especially noticed this in Audacity, where the entire application is laggy when the streamdeck service is running in the background (and foreground). There are other stability related things as well. Sometimes, when the app starts, it causes my system to become incredibly unstable, to the point where I have to restart to fix it. I primarily use my streamdeck for audio/video editing and this issue impacts my ability to do that well. Any advice helps.
![image](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/81998225/5609910f-2120-4eec-bcb8-03bd550d0c2b)
Stream Deck MK2

**Comment by Core447** [comment]
In fact `streamcontroller-linux-gui` is in maintaining mode ([more infos](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203)). The suggested alternative is [StreamController](https://flathub.org/apps/com.core447.StreamController) (I'm the developer of it 🙃). However, I can't guarantee that the performance will be better, but it's worth a try.

You can join the Discord [here](https://discord.gg/MSyHM8TN3u) if you want to.
